src/utils/hooks/useAxiosCategories.js

```diff
@@ -0,0 +1,41 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { API_BASE_URL } from "../constants";
+
+export function useAxiosCategories(query = "", page = 1) {
```

Comment:
are you planning on having all requests to the api with Axios or this one is just to test? Have you had any trouble with this?

Reply:
Yup I was testing to make it work with axios, but I was running out of time so I just committed a WIP since I did had troubles making it work.

Reply:
I made it work!